setup.py

```diff
@@ -37,6 +37,7 @@
         "Programming Language :: Python :: 3.3"
     ],
     install_requires=requires,
+    setup_requires=["cffi>=1.3.0"],
```

Comment:
Not sure what this does, but when it's present, it causes it to require a working C compiler (I think it's trying to compile cffi from scratch?) instead of using the premade wheels for cffi. With this removed, setup seems to succeed for me and no compilation is needed.